tests/integration/edge-handler.test.ts

```diff
@@ -261,27 +261,27 @@ describe("aborts middleware execution when the matcher conditions don't match th
 
     ctx.cleanup?.push(() => origin.stop())
 
-    for (const path of ['/hello', '/en/hello', '/nl-NL/hello', '/nl-NL/about']) {
+    for (const path of ['/hello', '/en/hello', '/nl-NL/about']) {
       const response = await invokeEdgeFunction(ctx, {
         functions: ['___netlify-edge-handler-middleware'],
         origin,
         url: path,
       })
-      expect(response.headers.has('x-hello-from-middleware-res'), `does match ${path}`).toBeTruthy()
+      expect(
+        response.headers.has('x-hello-from-middleware-res'),
+        `does not match ${path}`,
+      ).toBeTruthy()
       expect(await response.text()).toBe('Hello from origin!')
       expect(response.status).toBe(200)
     }
 
-    for (const path of ['/hello/invalid', '/about', '/en/about']) {
+    for (const path of ['/hello/invalid', '/invalid/hello', '/about', '/en/about']) {
       const response = await invokeEdgeFunction(ctx, {
         functions: ['___netlify-edge-handler-middleware'],
         origin,
         url: path,
       })
-      expect(
-        response.headers.has('x-hello-from-middleware-res'),
-        `does not match ${path}`,
-      ).toBeFalsy()
+      expect(response.headers.has('x-hello-from-middleware-res'), `does match ${path}`).toBeFalsy()
```

Comment:
These `expect` messages were the wrong way around, which was complicating the debugging somewhat!

Reply:
would writing message with "should" instead of "does (not)" make things less confusing? so in this case "middleware should not match ${path}" or something like that? then it doesn't matter wether it passes or not - intent is clear on what should happen

Reply:
updated in https://github.com/netlify/next-runtime/pull/2548/commits/fb8d9ea93cfdd41fa3fc2e5dcb9d787da818f8cf